app/services/milestone-constants.js

```diff
@@ -0,0 +1,88 @@
+import Service from '@ember/service';
+
+export default class MilestoneConstantsService extends Service {
+  LAND_USE_FEE_PAYMENT = '6a3beec4-dad0-e711-8116-1458d04e2fb8';
+
+  PREPARE_FILED_LAND_USE_APPLICATION = '663beec4-dad0-e711-8116-1458d04e2fb8';
+
+  CEQR_FEE_PAYMENT = '763beec4-dad0-e711-8116-1458d04e2fb8';
+
+  EIS_DRAFT_SCOPE_REVIEW = '7c3beec4-dad0-e711-8116-1458d04e2fb8';
+
+  FILED_EAS_REVIEW = '783beec4-dad0-e711-8116-1458d04e2fb8';
+
+  FINAL_SCOPE_OF_WORK_ISSUED = '823beec4-dad0-e711-8116-1458d04e2fb8';
+
+  EIS_PUBLIC_SCOPING_MEETING = '7e3beec4-dad0-e711-8116-1458d04e2fb8';
+
+  DEIS_PUBLIC_HEARING_HELD = '863beec4-dad0-e711-8116-1458d04e2fb8';
+
+  FEIS_SUBMITTED_AND_REVIEW = '883beec4-dad0-e711-8116-1458d04e2fb8';
+
+  NOC_OF_DRAFT_EIS_ISSUED = '843beec4-dad0-e711-8116-1458d04e2fb8';
+
+  REVIEW_SESSION_CERTIFIED_REFERRED = '8e3beec4-dad0-e711-8116-1458d04e2fb8';
+
+  COMMUNITY_BOARD_REFERRAL = '923beec4-dad0-e711-8116-1458d04e2fb8';
+
+  BOROUGH_PRESIDENT_REFERRAL = '943beec4-dad0-e711-8116-1458d04e2fb8';
+
+  BOROUGH_BOARD_REFERRAL = '963beec4-dad0-e711-8116-1458d04e2fb8';
+
+  CPC_PUBLIC_MEETING_PUBLIC_HEARING = '9e3beec4-dad0-e711-8116-1458d04e2fb8';
+
+  CPC_PUBLIC_MEETING_VOTE = 'a43beec4-dad0-e711-8116-1458d04e2fb8';
+
+  CITY_COUNCIL_REVIEW = 'a63beec4-dad0-e711-8116-1458d04e2fb8';
+
+  MAYORAL_VOTE = 'a83beec4-dad0-e711-8116-1458d04e2fb8';
+
+  FINAL_LETTER_SENT = 'aa3beec4-dad0-e711-8116-1458d04e2fb8';
+
+  upcoming = [
+    this.LAND_USE_FEE_PAYMENT,
+    this.PREPARE_FILED_LAND_USE_APPLICATION,
+    this.CEQR_FEE_PAYMENT,
+    this.EIS_DRAFT_SCOPE_REVIEW,
+    this.FILED_EAS_REVIEW,
+    this.FINAL_SCOPE_OF_WORK_ISSUED,
+    this.EIS_PUBLIC_SCOPING_MEETING,
+    this.DEIS_PUBLIC_HEARING_HELD,
+    this.FEIS_SUBMITTED_AND_REVIEW,
+    this.NOC_OF_DRAFT_EIS_ISSUED,
+    this.REVIEW_SESSION_CERTIFIED_REFERRED,
+    this.COMMUNITY_BOARD_REFERRAL,
+    this.BOROUGH_PRESIDENT_REFERRAL,
+    this.BOROUGH_BOARD_REFERRAL,
```

Comment:
Hmmm this works, right? Can't remember if `this` refers to the class scope, or the outer global scope in which the class itself is defined.

Reply:
Hmm, seems like it will work as long as I don't `new` the service, which I don't think is what Ember does during service injection? 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#Boxing_with_prototype_and_static_methods

Reply:
I was wondering the same thing, though